### PR TITLE
Adds Zendesk Destination function

### PR DESCRIPTION
**Pull request by misteryeo**
This specifically helps add new tickets to Zendesk based on Track actions taken.

### CODE REVIEW

File: destinations/zendesk/index.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+                   UserID: ${event.userId}
+                   Include steps here for what your Support Team to action on when seeing this application submitted`,
+          // This ensures that this is created as in internal note in Zendesk
+          "public": false
+        },
+        "priority": "high",
+        "tags": ["application-support"]
+      }
+    }
+    console.log('new ticket payload', newTicket)
```
Comment (markhazlett) [inline destinations/zendesk/index.js]: Remove console.log

File: destinations/zendesk/index.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
+async function track(event, settings) {
+  const endpoint = `https://${settings.subdomain}.zendesk.com/api/v2/tickets.json`
+  
+  if (event.event = "Application Submitted") {
+    
+    const newTicket = {
+      "ticket": {
+        "subject":  `Application Review: ${event.properties.display_name}`,
+        "comment":  { 
+          "body": `Type: ${event.properties.component_type}
```
Comment (markhazlett) [inline destinations/zendesk/index.js]: What's the reason to include these all in a string inside of the body? Any way we could add some details about how this shows up downstream?

Comment (misteryeo) [reply destinations/zendesk/index.js]: That's the only area you can include custom information - as an internal note in Zendesk.

File: destinations/zendesk/index.js (hunk 1 above)
Comment (misteryeo) [reply destinations/zendesk/index.js]: Fixed!

**Pull request merged**